### PR TITLE
Ana/balances v2 return 0 balance

### PR DESCRIPTION
**Pull request by Bitcoinera**
Was already merged as luniehq/lunie-api#900, but open here manually since this is now our official repo

### CODE REVIEW

**Comment by Bitcoinera** [comment]
Here it is good, it just ran the API checks. Why not the same in #4216 ?

**Comment by Bitcoinera** [comment]
Go away CircleCI!

File: api/.eslintrc.json [hunk 1]
```diff
@@ -14,5 +14,6 @@
       "ecmaVersion": 2018
   },
   "rules": {
+    "no-console": "off"
```
Comment (faboweb) [inline api/.eslintrc.json:17]: we probably should use one eslintrc for all repos

Comment (Bitcoinera) [reply api/.eslintrc.json:17]: yeah, good point!

**Pull request merged**